src/components/camera/server.cpp

```diff
@@ -0,0 +1,162 @@
+#include <components/camera/server.hpp>
+#include <components/camera/camera.hpp>
+#include <rpc/server.hpp>
+#include <gen/google/api/http.pb.h>
+
+::grpc::Status CameraServer::DoCommand(
+    ::grpc::ServerContext* context,
+    const ::viam::common::v1::DoCommandRequest* request,
+    ::viam::common::v1::DoCommandResponse* response) {
```

Comment:
I noticed that in other methods, we check that the `request` is non-null and return an error otherwise. Can we do the same here?